Edits/azure-service-bus-management-connector-studio.adoc

```diff
@@ -0,0 +1,135 @@
+
+* `HTTP Listener`, which initiates a flow each time it receives a request on the configured host and port
+* `Scheduler`, which initiates a flow when a time-based condition is met
+
+For example, to configure `HTTP Listener`, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Azure Service Bus Management Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Azure Service Bus Management* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Azure Service Bus instance. Azure Service Bus Management Connector supports Shared Access signature (SAS) authentication.
+
+To configure the global element for Azure Service Bus Management Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, configure Shared Access Signature (SAS) authentication by entering values for these fields:
++
+[%header,cols="30s,70a"]
+|===
+|Field |User Action
+|Service Namespace |Namespace for the Azure Service Bus Service within your subscription
+|Shared Access Key Name | Name of your SAS token
+|Shared Access Key | SAS token
+|===
++
+The following screenshot shows an example of configuring Shared Access Signature (SAS) authentication. This screenshot uses property placeholders in place of field values. For information on using property placeholders, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
```

Comment:
GLOBALLY lowercase SAS spelling.
Reworded to avoid "placeholders in place of."
```suggestion
The following screenshot shows an example of configuring shared access signature (SAS) authentication. This screenshot uses property placeholders for field values. For information about using property placeholders, see Anypoint Connector Configuration.
```
Do we really need the xref?
If so, move to See Also.